vhdbuilder/packer/vhd-scanning.sh

```diff
@@ -19,6 +19,12 @@ VHD_IMAGE="$MANAGED_SIG_ID"
 SIG_CONTAINER_NAME="vhd-scans"
 SCAN_VM_ADMIN_USERNAME="azureuser"
 
+RELEASE_NOTES_FILEPATH="$(pwd)/release-notes.txt"
```

Comment:
could we just create the file here instead of erroring out? Or is that a sign of some other issue?

Reply:
we expect release note to exist at this point, from previous packer build step